website/docs/r/managed_disk.html.markdown

```diff
@@ -97,6 +97,8 @@ The following arguments are supported:
 
 ~> **NOTE:** Disk Encryption Sets are in Public Preview in a limited set of regions
 
+-> **NOTE:** Encryption type of the key will be decided by the disk encryption set. [More info on encryption type](https://docs.microsoft.com/en-us/azure/virtual-machines/disk-encryption)
```

Comment:
this fields configured on the Disk Encryption Set rather than the VM, so this comment isn't really necessary:

```suggestion
```